interface/minio_api.py

```diff
@@ -0,0 +1,35 @@
+import io
+from minio import Minio
+from django.conf import settings
+
+
+class MinioAPI:
+    __instance = None
+
+    @staticmethod
+    def getInstance():
```

Comment:
Ah, the Java singleton pattern :) You could just write `minio_api = MinioAPI()` at the bottom of the file and you have your singleton importable by anyone ;) Even better, get rid of this class, and make `client = Minio(...)` into a module-level object and convert `upload` and `download` to plain functions instead of methods.